### DISCUSSION TITLE
Filter icon

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#775] [Request: Icon] [answered]
Hello,
We are working on a IoT Super SIM filter feature. I'm wondering, if we can have a new icon for Filter. If we don't, can I use the react icon which is sunsetting?

**Comment by corinne-nagel** [comment, marked as answer]
Hi @ishah-twilio when would you be needing this icon by?!

**Reply by ishah-twilio** [reply]
November 2020.

**Reply by corinne-nagel** [reply]
Ok cool, looks like you have a little time! I've added a ticket here:
https://issues.corp.twilio.com/browse/DSYS-2284

For next time, we'd love it if you could contribute yourself!
https://paste.twilio.design/icons/how-to-add-an-icon

**Comment by serifluous** [comment]
Hey @ishah-twilio ! The Filter icon is available in Paste now: https://paste.twilio.design/icons/ You'll notice that the glyph looks different from what's currently in the products—We decided to change the shape since the original shape loses clarity at smaller sizes.

The icon isn't available yet in Sketch, though. We're going to merge that in during our next release (Nov 10).
